config/application.rb

```diff
@@ -59,7 +61,5 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
```

Comment:
You'll need to add `lib` to `eager_load_paths` in here to ensure `lib` is loaded when in production env (as per https://github.com/alphagov/manuals-frontend/pull/247) because https://blog.bigbinary.com/2016/08/29/rails-5-disables-autoloading-after-booting-the-app-in-production.html

Reply:
:+1: Updated